### PR TITLE
Safari TP89 is out, with ?? operator suport

### PR DESCRIPTION
**Pull request by afmenez**
https://webkit.org/blog/9497/release-notes-for-safari-technology-preview-89/

### CODE REVIEW

**Review by ljharb — APPROVED** [review]

**Review by chicoxyzzy — CHANGES_REQUESTED** [review]
Nullish coalescing is not enabled by default in TP89. See https://twitter.com/rkirsling/status/1159568652397502464

**Comment by afmenez** [comment]
Changed the result to "flagged" and added a note on enabling it.

**Review by chicoxyzzy — APPROVED** [review]
LGTM. Though I'd change note wording a bit:
The feature have to be enabled via JSC runtime flag `__XPC_JSC_useNullishCoalescing=true`

File: data-esnext.js [hunk 1]
```diff
@@ -1712,6 +1712,7 @@ exports.tests = [
     babel7corejs2: true,
     ie11: false,
     firefox52: false,
+    safaritp: {val: 'flagged', note_id: "safari-nullish", note_html: "You can flip it on like this: <code>__XPC_JSC_useNullishCoalescing=true && open -a 'Safari Technology Preview'</code>."},
```
Comment (ljharb) [inline data-esnext.js]: no &&, or it won’t work
```suggestion
    safaritp: {val: 'flagged', note_id: "safari-nullish", note_html: "You can flip it on like this: <code>__XPC_JSC_useNullishCoalescing=true open -a 'Safari Technology Preview'</code>."},
```

Comment (afmenez) [reply data-esnext.js]: Done

**Review by ljharb — APPROVED** [review]

**Pull request merged**